server/global-variables.d.ts

```diff
@@ -53,4 +56,5 @@ declare global {
 	const Users: typeof UsersType.Users;
 	const Verifier: typeof VerifierType;
 	const toID: typeof DexType.toID;
+	const __fsState: {pendingUpdates: Map<string, PendingUpdate>};
```

Comment:
I would actually explicitly not declare this here; since the entire point of not using `FS` as a global is so that it's not used outside the FS package.